src/api/app/controllers/statistics/maintenance_statistics_controller.rb

```diff
@@ -1,12 +1,24 @@
 module Statistics
   class MaintenanceStatisticsController < ApplicationController
-    skip_before_action :extract_user
+    skip_before_action :extract_user, :require_login
```

Comment:
That's already handled. Though I kept the existing route as well, to stay compatible